synapse/lib/storm.py

```diff
@@ -2187,6 +2187,14 @@ def allowed(self, perms, gateiden=None, default=None):
 
         return self.user.allowed(perms, gateiden=gateiden, default=default)
 
+    def reqEditProp(self, prop):
+
+        if len(prop.setperms) > 1:
+            if any(self.allowed(perm, gateiden=self.snap.wlyr.iden) for perm in prop.setperms[:-1]):
+                return
+
+        self.confirm(prop.setperms[-1], gateiden=self.snap.wlyr.iden)
```

Comment:
If the list is reversed at Prop construction time, this index becomes `[0]` to get the new style perm in the authdeny message.

Reply:
need to check allowed on the others and then confirm on this one last and a `0` index lookup is not faster than a `-1` index lookup. If they weren't in this order, i'd need to reverse the slicing logic above anyway. I think the current way is most optimal.